### PR TITLE
Switch to store in userStroies

### PR DESCRIPTION
**Pull request by alshakerM**
Changed how we get the data in stories from React.useEffect fetch to store.

### CODE REVIEW

**Comment by alshakerM** [comment]
Changed how we get the data in suggestions from React.useEffect fetch to store.

File: src/storiesStore/reducer.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  stories: [],
+};
+
+export default function reducer(state = defaultState, action) {
+  switch (action.type) {
+    case 'RECEIVE_STORIES': {
+      return { ...state, stories: action.stories };
+    }
+    case 'SET_LOADING': {
+      const newState = { ...state, stories: [...state.stories] };
```
Comment (alshakero) [inline src/storiesStore/reducer.js:11]: You're not mutating stories. 
```suggestion
      const newState = { ...state };
```

File: src/storiesStore/selectors.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export function getStories(state) {
+  return state.stories;
+}
+export function getIsLoading(state) {
+  return state.isLoading;
+}
```
Comment (alshakero) [inline src/storiesStore/selectors.js:6]: This is nice and let's leave it. But try to only add selectors as you need them.

**Review by alshakero — APPROVED** [review]
LGTM! Two tiny issues.

File: src/storiesStore/reducer.js (hunk 1 above)
Comment (alshakerM) [reply src/storiesStore/reducer.js:11]: On it.

**Pull request merged**